tidy3d/components/base.py

```diff
@@ -22,9 +23,12 @@
 from ..exceptions import FileError
 from ..log import log
 
-# default indentation (# spaces) in files
-INDENT = 4
+
+INDENT_JSON_FILE = 4  # default indentation of json string in json files
```

Comment:
we could consider setting this to 2 to reduce the size of the file itself, but no preference either way.

Reply:
note: I'm not sure this will actually change the file size. depends on how they json handles indents I guess.

Reply:
Tested. It does change the size. I guess let's do 2 yeah? But this shouldn't matter for upload/download from the server because all this is done through hdf5 files now. So not as important potentially, could be left 4 since it's prettier?

Reply:
fine by me. I prefer 4 if the file size is not an issue.